src/components/Search.jsx

```diff
@@ -17,8 +16,47 @@ const Search = () => {
         />
       </div>
       <div className="flex justify-evenly w-auto ml-4 space-x-2">
-        <FilterIcon />
-        <SortIcon />
+        {/* Filter  */}
+        <button>

```

Comment:
The expectation is after you export (ONLY) svgs icons from the `icons.jsx`, you can then use them in here. So typically the svg here should be replaced.

Applies every other place as well.

Reply:
Oh okay. Lets leave it for now.